### DISCUSSION TITLE
Filter variants by HPO related gene lists

### DISCUSSION THREAD

**Discussion by AlessandroLas** [broadinstitute/seqr#3163] [Q&A]
Hi all.

Is it possible to filter variants by HPO related gene lists?
Do one have to create them manually?
The "Features" field of the family page allows adding (or excluding) HPO terms but it is unclear
if the selected HPO terms are useful for variant prioritization (or filtering).

Many thanks in advance for your help.

**Comment by hanars** [comment]
Currently you need to either manually create gene lists for HPO terms of interest, or seqr has an integration with [PanelApp](https://panelapp.genomicsengland.co.uk/panels/) and you can use their curated gene panels as disease/phenotype specific gene lists (although there is no direct mapping from PanelApp to HPO)

The HPO terms displayed in seqr are mainly used by analysts qualitatively, either to select a gene list of interest at search time or to assess whether a candiate gene/variant has a good phenotypic match for the case

**Comment by AlessandroLas** [comment]
Thanks Hana.

I've been creating HPO lists manually...
Can you indicate where does seqr store those lists?
I'd like to create a backup copy of the file or also to share it.

**Reply by hanars** [reply]
I am not sure what you mean by "HPO lists", but if you mean you have been adding HPO terms to individuals in seqr, you can use the "Download Table" button on the project page to download all the individuals and that will include their HPO terms

**Reply by AlessandroLas** [reply]
It was related to the discussion.
The question is...
Where does seqr store manually created gene lists? 
Is there a "command line" way to get the manually created gene lists without going to the web interface?
Thanks.

**Reply by hanars** [reply]
seqr is not a command line tool. There is no supported seqr functionality available outside of the web interface. If you want to download gene lists, this functionality is available in the web application by clicking "Summary Data" in the page header, selecting "Gene Lists", selecting the list of interest, and then clicking the "Download" button
